spec/ic.did

```diff
@@ -58,6 +58,8 @@ service ic : {
       module_hash: opt blob;
       memory_size: nat;
       cycles: nat;
+      freezing_threshold: nat;
+      freezing_threshold_in_cycles: nat;
```

Comment:
Maybe I misunderstand, but related to the above, does this belong into the settings record?

Reply:
No, this is not something that can be set but it's computed by the IC based on the `freezing_threshold`. Also, there was a follow-up PR that was merged so it's not called `freezing_threshold_in_cycles` anymore.